packages/devtools_app/lib/src/memory/memory_snapshot_models.dart

```diff
@@ -169,6 +208,10 @@ class Reference extends TreeNode<Reference> {
 
   final String name;
 
+  final bool isAllocation;
```

Comment:
document the difference between isAllocation and isAllocations.

Reply:
/// Allocations are being monitored, user requested monitoring of any
  /// allocated memory.  Parent node containing all monitored classes.
  final bool isAllocations;

  /// A monitored class.
  final bool isAllocation;